Authenticator/Source/AppController.swift

```diff
@@ -43,8 +43,15 @@ class AppController {
     }()
 
     init() {
+        if Process.isDemo {
+            store = DemoTokenStore()
```

Comment:
I also have a test branch that uses the memory store and mocks out some UITableView stuff.